Ecosystem Partners/Marketplace Guides/getting-started-with-cloudberry-lab-backup-enterprise-blueprint.md

```diff
@@ -0,0 +1,191 @@
+
+### Prerequisite
+- Access to the CenturyLink Cloud platform as an authorized user.
+- Existing Windows Platform server to deploy to
+- Configure the VPN access to the virtual server as it is described [at this link](https://www.ctl.io/knowledge-base/network/how-to-configure-client-vpn/).
+
+### Deploy CloudBerry Labs Cloud Storage Tools Blueprint
+Follow these step by step instructions to deploy Cloudberry Backup Enterprise Edition on your server and connect to you Centurylink Object Storage account. 
+
+1.	Open the Blueprint Library.
+- Access your Blueprint Library by selecting the “Blueprints Library” under the Blueprints heading from the green drop down menu.
+
+  [CloudBerry Image](../../images/ecosystem-CloudBerry-1.png)
+
+2.	Search for the Blueprint.
+- To search for the Nexus Chameleon blueprint, type “Cloudberry Backup Enterprise for Desktop and Server” under “Refine Results” in the right panel and click the “Go” button.
+
+  [CloudBerry Image](../../images/ecosystem-CloudBerry-2.png)
+
+3.	Choose the Blueprint. 
+- Click on the Blueprint titled “CloudBerry Backup Enterprise for Desktop and Server”
+
+  [CloudBerry Image](../../images/ecosystem-CloudBerry-3.png)
+
+4.  Deploy the Blueprint.
+- Click on the “Deploy Blueprint” button to begin configuring your installation.
+
+  [CloudBerry Image](../../images/ecosystem-CloudBerry-4.png)
+
+5.	Configure the Blueprint. 
+- On the first page, “Customize Blueprint”, ensure the following options are configured:
+  - Choose the server, you would like to install Cloudberry Backup Enterprise on.
+  - Enter the e-mail, which you would like to use for trial activation.
+
+  [CloudBerry Image](../../images/ecosystem-CloudBerry-5.png)
+
+  - Then select "Next Step"
+
+  [CloudBerry Image](../../images/ecosystem-CloudBerry-6.png)
+
+6.	Review and Confirm the Blueprint. 
+- You will come to a confirmation view of what your Blueprint looks like.
+- Verify your configuration details.
+- Then click the green “deploy Blueprint” button at the bottom.
+
+  [CloudBerry Image](../../images/ecosystem-CloudBerry-7.png)
+
+7.	Monitor the Activity Queue.
+- After clicking Deploy Blueprint, the job will be submitted into a queue and you will be taken to a monitoring page where you can see the progress of each step the Blueprint goes through.  
+
+  [CloudBerry Image](../../images/ecosystem-CloudBerry-8.png)
+
+- You can access the queue at any time by clicking the Queue link under the Blueprints menu on the main navigation drop-down.
+
+  [CloudBerry Image](../../images/ecosystem-CloudBerry-9.png)
+
+6.	Jump In! 
+- Once the Blueprint completes successfully, you will receive an email stating that the Blueprint build is complete. Please do not use the application until you have received this email notification. 
+
+### Access and use Nexus Chameleon
```

Comment:
Thanks Chris, fixing.


Bob Stolzberg | Principal Engineer
CenturyLink
Mobile/Text: (314) 397-2610<tel:(314)%20397-2610>
E-Mail: bob@CTL.io<mailto:bob@CTL.io>




On Sep 30, 2015, at 1:36 PM, Chris Little <notifications@github.com<mailto:notifications@github.com>> wrote:


In Ecosystem Partners/Marketplace Guides/getting-started-with-cloudberry-lab-backup-enterprise-blueprint.md<https://github.com/CenturyLinkCloud/PublicKB/pull/457#discussion_r40832749>:

> +
> +  [CloudBerry Image](../../images/ecosystem-CloudBerry-7.png)
> +
> +7.   Monitor the Activity Queue.
> +- After clicking Deploy Blueprint, the job will be submitted into a queue and you will be taken to a monitoring page where you can see the progress of each step the Blueprint goes through.
> +
> +  [CloudBerry Image](../../images/ecosystem-CloudBerry-8.png)
> +
> +- You can access the queue at any time by clicking the Queue link under the Blueprints menu on the main navigation drop-down.
> +
> +  [CloudBerry Image](../../images/ecosystem-CloudBerry-9.png)
> +
> +6.   Jump In!
> +- Once the Blueprint completes successfully, you will receive an email stating that the Blueprint build is complete. Please do not use the application until you have received this email notification.
> +
> +### Access and use Nexus Chameleon


Nexus chameleon?

-
Reply to this email directly or view it on GitHub<https://github.com/CenturyLinkCloud/PublicKB/pull/457/files#r40832749>.

This communication is the property of CenturyLink and may contain confidential or privileged information. Unauthorized use of this communication is strictly prohibited and may be unlawful. If you have received this communication in error, please immediately notify the sender by reply e-mail and destroy all copies of the communication and any attachments.